bundles/python/misc/examples.py

```diff
@@ -0,0 +1,252 @@
+#! /usr/bin/env python
+
+# Comments use '#'
+
+# Lists
+
+empty_list = []
+
+# Builtin Identifiers
+
+builtins = [ArithmeticError, AssertionError, AttributeError,
+BaseException, BufferError, BytesWarning, DeprecationWarning,
+EOFError, Ellipsis, EnvironmentError, Exception, False,
+FloatingPointError, FutureWarning, GeneratorExit, IOError,
+ImportError, ImportWarning, IndentationError, IndexError, KeyError,
+KeyboardInterrupt, LookupError, MemoryError, NameError, None,
+NotImplemented, NotImplementedError, OSError, OverflowError,
+PendingDeprecationWarning, ReferenceError, RuntimeError, RuntimeWarning,
+StandardError, StopIteration, SyntaxError, SyntaxWarning, SystemError,
+SystemExit, TabError, True, TypeError, UnboundLocalError,
+UnicodeDecodeError, UnicodeEncodeError, UnicodeError,
+UnicodeTranslateError, UnicodeWarning, UserWarning, ValueError,
+Warning, ZeroDivisionError, __debug__, __doc__, __import__,
```

Comment:
Yes that was intentional since I wasn't sure if styling these is useful. But using `special` for all (built-in as well as user defined) double underscore identifiers seems like a good idea.